lib/hstcalversion.c

```diff
@@ -14,10 +14,10 @@ char * sprintfGitInfo(char ** buffer)
 
     const char * format = "git tag: %s\ngit branch: %s\nHEAD @: %s";
     size_t length = strlen(format) + strlen(VERSION) + strlen(BRANCH) + strlen(COMMIT); // NOTE: this doesn't require an explicit +1 for '\0' as it is larger than needed due to '%s' & '\n'.
-    *buffer = malloc(length*sizeof(char));
+    *buffer = malloc(length + 1 * sizeof(char));
```

Comment:
You want to do the `+` first, right?
```suggestion
    *buffer = malloc((length + 1) * sizeof(char));
```

Reply:
I can express it this way for clarity, but the compiler will evaluate it the same way with/without the parenthesis.